### DISCUSSION TITLE
Open OnlyOffice file in Brave Browser

### DISCUSSION THREAD

**Discussion by foxhoundv** [nextcloud/all-in-one#3224] [Questions] [answered]
I don't know exactly where the problem lies, whether its AIO or the browsers fault or maybe both. I use Brave Browser and a friend was telling me they couldn't access any .docx, .xlsx file in the browser. Which they use Brave as well. I checked from my browser which wasn't updated yet, and everything would open fine. Once I updated the browser, now it won't open files and just automatically tries to download the file instead. Anyone else getting this issue with maybe a workaround?

EDIT: I just checked with Google Chrome browser and it does the same thing.

**Comment by szaimen** [comment]
Hi, does it work in Firefox?

**Reply by foxhoundv** [reply]
No, it doesn't work in Firefox. It automatically downloads instead of opening it in the browser.

**Comment by szaimen** [comment, marked as answer]
Hi, please go to `https://yourdomain.com/settings/admin/onlyoffice` and simply click on the `Save` button which should start a test and enable the app. The fix will be done in https://github.com/nextcloud/all-in-one/pull/3232

**Reply by foxhoundv** [reply]
Awesome, that got it working again.

**Reply by szaimen** [reply]
This is now released with v7.1.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by foxhoundv** [reply]
I have been testing this for about a week on my secondary test server with no data on it and everything has been working fine.
